install/kubernetes/helm/istio/charts/ingress/templates/deployment.yaml

```diff
@@ -32,8 +32,10 @@ spec:
           - ingress
           - -v
           - "2"
-          - --discoveryAddress
+          - --discoveryPlainAddress
```

Comment:
wouldn't it be better to keep the previous option (--discoveryAddress) and just add the --discoveryMtlsAddress option? that way folks could keep the previous templates.

Reply:
+1  - no backward incompatible changes please.